torch/fx/graph_module.py

```diff
@@ -179,6 +183,41 @@ def graph(self, val) -> None:
         self._graph = val
         self.recompile()
 
+    def to_folder(self, folder, module_name="FxModule"):
+        folder = Path(folder)
+        Path(folder).mkdir(exist_ok=True)
+        torch.save(self.state_dict(), folder / 'state_dict.pt')
+        tab = " " * 4
+        module_str = f"""
+import torch
+class {module_name}(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        state_dict = torch.load('{folder}/state_dict.pt')
+"""
```

Comment:
Wish we had something like this: https://github.com/pytorch/pytorch/blob/0125e14c9a7cdd48f3d9c92447170c02d13ed24e/tools/codegen/code_template.py in PT itself